boards/arm/raspberrypi_pico/raspberrypi_pico.dts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2021 Yonatan Schachter
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <raspberrypi/rp2040.dtsi>
```

Comment:
I'd rather use `rpi` everywhere for the base name, and then add `-pico` when necessary. i.e. replace `raspberry` with `rpi` everywhere.